docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: '3'
+services:
+
+  #PHP Service
```

Comment:
Lets kill all the comments in this file. I don't think they add anything when its clear the app vs webserver vs db.